### DISCUSSION TITLE
🖼️ Mermaid.js for simplified chart integration

### DISCUSSION THREAD

**Discussion by OCram85** [thuliteio/doks#225] [Ideas]
### 📦 Content

I really like explaining complex technical facts with charts in docs.
Therefore [mermaid](https://mermaid-js.github.io/mermaid/#/) would work with this theme perfectly. 

>  **☄️ NOTE:** There could be race conditions with highlight.js and mermaid when rendering the code blocks 🚗  

#### 🔖 Ref

- #193

**Comment by C0DK** [comment]
Isn't this solved? i can see a bunch of references to it in the code, and it seemed to have been there in the docs, but is now removed... (the page is now a draft?) https://github.com/thuliteio/getdoks.org/blob/main/content/docs/built-ins/diagrams.md

@h-enk

**Reply by h-enk** [reply]
"At times, the upstream kroki.io service may run into resource exhaustion.
This can be due to a high volume of requests or a temporary outage."

See also: https://github.com/thuliteio/doks/pull/1266

I'm considering discontinuing support for the Kroki service and using Hugo's solution for [Mermaid diagrams](https://gohugo.io/content-management/diagrams/#mermaid-diagrams) support instead.
